src/main/java/com/corundumstudio/socketio/SocketIOServer.java

```diff
@@ -246,6 +242,11 @@ public void addConnectListener(ConnectListener listener) {
         mainNamespace.addConnectListener(listener);
     }
 
+    @Override
+    public void addPingListeners(PingListener pingListener) {
```

Comment:
@lovebing Please rename method to `addPingListener` since you're adding only one listener at a time